VMD_scripts/Blob_GUI.tcl

```diff
@@ -32,30 +56,62 @@ variable blobs [toplevel ".blob"]
 }
 proc ::blobulator::GUI {} {
 	::blobulator::Window
-	grid [label $::blobulator::blobs.1_MolID -text MolID ]
-	grid [entry $::blobulator::blobs.tv_MolID -width 10 -textvariable ::blobulator::MolID ] -row 0 -column 1
+	grid [label $::blobulator::blobs.1_MolID -text MolID: ] -row 0 -column $::blobulator::textColumn -sticky e
+	grid [entry $::blobulator::blobs.tv_MolID -width 10 -textvariable ::blobulator::MolID ] -row 0 -column $::blobulator::paraColumn -columnspan 1 
 	if {$::blobulator::MolID == ""} {
 		set ::blobulator::MolID "top"
 	}
-	set paraList [list Lmin ::blobulator::Lmin 1 50 1 H ::blobulator::H .1 1 .01]
-	foreach { entry variableName min max interval} $paraList {
-		set w1 [label $::blobulator::blobs.l_$entry -text $entry]
-		set w2 [entry $::blobulator::blobs.e_$entry -width 10 -textvariable $variableName]
-		set w3 [scale $::blobulator::blobs.s_$entry -orient horizontal -from $min -to $max -length 175 -resolution $interval -tickinterval 0 -variable $variableName -showvalue 0]
+
+	#Atomselect grids
+	grid [label $::blobulator::blobs.lselect -text "         Selection:" ] -row 0 -column $::blobulator::text2Column -sticky e
```

Comment:
You're addressing a lot of disparate issues in this PR. Which issue are these changes addressing?

Reply:
This issue was adressing UI changes. The spaces were to push the text further in the grid to align with parallel text.